package.json

```diff
@@ -14,13 +14,15 @@
     "@types/dotenv-safe": "8.1.2",
     "bcrypt": "5.1.0",
     "cookie": "0.5.0",
+    "dompurify": "^2.4.0",
```

Comment:
we use locked versions, since it's a bit more secure and we also have Renovate to keep the versions up to date

I'll take this over

Reply:
done in 552f4e0